src/Umbraco.Tests.AcceptanceTest/cypress/integration/Content/content.ts

```diff
@@ -326,8 +330,8 @@ context('Content', () => {
         cy.reload();
 
         // Assert
-        cy.get('.history').find('.umb-badge').eq(0).should('contain.text', "Save");
-        cy.get('.history').find('.umb-badge').eq(1).should('contain.text', "Rollback");
+        cy.get('.history').find('.umb-badge').eq(0).should('contain.text', "Rollback");
```

Comment:
When I did the tests I have manually tried Rolling back content and on the audit trail, I was always the "Save" label before the "Rollback" one.  When doing the tests sometimes it happened that the operation was too fast and "Rollback" was coming first (differing from what I observed during the manual reproduction steps), that's why I am reloading the page before asserting. And then after running the test a few times the ordering was as expected. So I am not sure about changing it as I would think that the typical scenario would be to rollback and then save the changes. 

cc @bergmania

Reply:
Hmm that's very strange, when I manually try and roll back I see the rollback label before the save label like in the screenshot, and likewise when I run the test.
It would make sense to me if it did the rollback and then saved it hpwever, meaning the Save should be before the rollback.

Reply:
Coming back now close to a month later it's back to being save first then rollback, this is an odd one.

I spoke to Niels a while ago and we talked about how it, in reality, should just be rollback only, since it's kind of confusing that it's both save and rollback since that suggests that you've performed two actions, but that's a separate issue really. I'm resolving this conversation for now :)